src/components/app/routes/data/utils.js

```diff
@@ -47,58 +47,80 @@ export async function ensureEnterpriseAppData({
   queryClient,
   requestUrl,
 }) {
+  const enterpriseAppDataQueries = [];
   const subscriptionsQuery = querySubscriptions(enterpriseCustomer.uuid);
-  const enterpriseAppDataQueries = [
-    // Enterprise Customer User Subsidies
-    queryClient.ensureQueryData(subscriptionsQuery).then(async (subscriptionsData) => {
```

Comment:
The `queryClient.ensureQueryData(subscriptionsQuery).then(...)` is now conditional based on which route the user is currently on and/or navigating to. If a BFF query is configured for the current page route, the existing fetching of `learner-licenses` directly is skipped in favor of relying on the BFF query instead (which should fetch the same data and perform the side effects as below).